### PR TITLE
Update Python version to 3.11

### PR DESCRIPTION
**Pull request by SpeedOfMagic**

### CODE REVIEW

**Review by IINemo — APPROVED** [review]
Looks good, as long as the code passes all the tests.

**Comment by IINemo** [comment]
Could you also update Python version in the Readme?

**Comment by SpeedOfMagic** [comment]
Launched local tests on colab, test_coqa_instruct and test_persion_bio failed because of low memory on GPUs
=> I think upgrading to Python 3.11 works.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://github.com/IINemo/isanlp_srl_framebank/blob/master/LICENSE)
 ![Python 3.10](https://img.shields.io/badge/python-3.10-blue.svg)
+![Python 3.11](https://img.shields.io/badge/python-3.11-blue.svg)
```
Comment (SpeedOfMagic) [inline README.md:3]: Done to show that lm-polygraph supports both python 3.10 and python 3.11

**Pull request merged**